features/git-town/configuration/configuration_automatic.feature

```diff
@@ -27,13 +27,31 @@ Feature: Automatically running the configuration wizard if Git Town is unconfigu
     | git sync-fork      |
 
 
+  Scenario Outline: Proceeding to configuration upon initial config prompt (using prompt's default)
+    When I run `<COMMAND>` and enter "" and "^C"
+    Then I see "Git Town hasn't been configured for this repository."
```

Comment:
I'm not sure it's unnecessary, as there are 2 implicit steps here. 

1. the user types a git town command, hits enter, then sees an initial config prompt
2. then the user types "y" or "n" or "", hits enter, then sees the first question of the config

Buuut as I type this now, I see that it should probably be split up huh? One scenario outline that tests for the "Git Town hasn't been configured..." message. Another scenario outline that does what this one here does